pkg/test/framework/features/features.yaml

```diff
@@ -61,15 +61,51 @@ features:
     user:
     ingress:
       mtls:
+        server-keyrotation:
+        generic-compoundrotation:
+        nongeneric-compoundrotation:
+        gateway:
+          invalid-secret:
+          valid-secret:
+          K8sca:
       tls:
+        secretrotation:
+        gateway:
+          invalid-secret:
+          K8sca:
     egress:
       mtls:
         sds:
       tls:
         filebased:
         sds:
+    vault:
+      sds:
+    authorization:
```

Comment:
@yangminzhu could you please help to review the policy feature labels?

Reply:
The following seems just a copy of the tests which is not useful in deciding what is covered or not, otherwise you will find everything is covered because the feature list is the same as the test list.

Instead, let's take a look at the authorization policy fields and add each field to this list, we will also need to include meaningful combinations of different fields and values, for example:
```
selector
  namespace-level (empty selector)
  workload-level (1 key/values)
  workload-level (N key/values)
from
  1 rule
  2 rules (OR-ed together)
operation
  path with exact/suffix/prefix/exist match
...
```

Reply:
@JimmyCYJ @williamaronli  can you explain a little bit what exactly we should put here? the current list is just an exact copy of all the test cases, I honestly don't know how could it be useful.